upload-api/functions/ucan-invocation-router.js

```diff
@@ -59,10 +59,11 @@ async function ucanInvocationRouter (request) {
 
   const { UPLOAD_API_DID } = process.env;
   const { PRIVATE_KEY } = Config
-  const serviceSigner = getServiceSigner({ UPLOAD_API_DID, PRIVATE_KEY })
+  const servicePrincipal = getServerPrincipal({ UPLOAD_API_DID, PRIVATE_KEY })
+  const serviceSigner = getServiceSigner({ PRIVATE_KEY })
   const ucanStoreBucket = createUcanStore(AWS_REGION, ucanBucketName)
 
-  const server = await createUcantoServer(serviceSigner, {
+  const server = await createUcantoServer(servicePrincipal, {
```

Comment:
Notably, the `serviceSigner` (no env.did) is still used with `createAccessClient` below